molecule/default/yamllint.yml

```diff
@@ -1,3 +1,4 @@
+# SPDX-License-Identifier: MIT
 ---
 extends: default
 rules:
```

Comment:
Please see https://github.com/linux-system-roles/timesync/pull/47 for further change to be made